openlibrary/catalog/marc/tests/test_marc_html.py

```diff
@@ -0,0 +1,24 @@
+from openlibrary.catalog.marc.html import html_record
+
+
+def test_html_subfields():
+    samples = [
+        ('  \x1fa0123456789\x1e', '<b>$a</b>0123456789'),
+        ('  end of wrapped\x1e', 'end of wrapped'),
+        ('  \x1fa<whatever>\x1e', '<b>$a</b>&lt;whatever&gt;'),
+    ]
+    hr = html_record("00053This is the leader.Now we are beyond the leader.")
+    for input, output in samples:
+        assert hr.html_subfields(input) == output
+
+
+def test_html_line():
```

Comment:
This test fails because the encoding (marc8 or utf8) is not defined for a single field, and the code seems to default to picking marc8. There isn't a quick way to fix this properly, #3057 already exists to bring this code up to standard.

Marking it with a pytest `@pytest.mark.xfail` will indicate it's a known issue that needs to be addressed in #3057